api/storage/client.go

```diff
@@ -49,6 +50,30 @@ func (c *Client) Show(tags []names.StorageTag) ([]params.StorageInstance, error)
 	return all, allErr.Combine()
 }
 
+// PoolList lists pools according to a given filter.
+// If no filter was provided, this will return a list
+// of all pools.
+func (c *Client) PoolList(types, names []string) ([]params.PoolInstance, error) {
+	args := params.PoolListFilter{
+		Names: names,
+		Types: types,
+	}
+	found := params.PoolListResults{}
```

Comment:
"Results" is usually used for bulk method calls, i.e. where there are multiple disparate arguments and corresponding results. As such, this should just be "PoolListResult". I think it should just be "PoolsResult" though.